src/js/pages/services/DeploymentsTab.js

```diff
@@ -1,11 +1,73 @@
+import moment from 'moment';
 import React from 'react';
+import {Table} from 'reactjs-components';
 
 import AlertPanel from '../../components/AlertPanel';
+import EventTypes from '../../constants/EventTypes';
+import MarathonStore from '../../stores/MarathonStore';
+import ResourceTableUtil from '../../utils/ResourceTableUtil';
+import StringUtil from '../../utils/StringUtil'
+
+const columnClassName = ResourceTableUtil.getClassName;
+const columnHeading = ResourceTableUtil.renderHeading({
+  id: 'DEPLOYMENT',
+  startTime: 'STARTED',
+  status: 'STATUS'
+});
+const columns = [
+  {
+    className: columnClassName,
+    heading: columnHeading,
+    prop: 'id',
+    render: function (prop, deployment) {
+      return deployment.getId();
+    }
+  },
+  {
+    className: columnClassName,
+    heading: columnHeading,
+    prop: 'startTime',
+    render: function (prop, deployment) {
+      return moment(deployment.getStartTime()).fromNow();
+    }
+  },
+  {
+    className: columnClassName,
+    heading: columnHeading,
+    prop: 'status',
+    render: function (prop, deployment) {
+      return `Step ${deployment.getCurrentStep()} of ${deployment.getTotalSteps()}`;
+    }
+  }
+];
 
 class DeploymentsTab extends React.Component {
 
   constructor() {
     super(...arguments);
+    this.onDeploymentsChange = this.onDeploymentsChange.bind(this);
+
+    this.state = {
+      deployments: MarathonStore.get('deployments')
+    };
+  }
+
+  componentDidMount() {
+    MarathonStore.addChangeListener(
```

Comment:
Not part of this PR, but it would be cool, if we could get the data from the DCOStore instead, to have components only deal with one unified store.